src/Campaigns/resources/admin/components/CampaignDetailsPage/Components/TextareaControl/styles.scss

```diff
@@ -0,0 +1,29 @@
+.givewp-textarea-control.givewp-textarea-control.givewp-textarea-control {
```

Comment:
why?? haha

Reply:
![CleanShot 2025-02-18 at 13 00 06](https://github.com/user-attachments/assets/e5773b60-bf2d-4a0e-af7a-bdad2e9ed740)

Overwriting styles without `!important`. 🤣

Reply:
I currently don’t see any other textarea fields in the Settings. Do you think it is safe to remove that rule, which is being generically applied to textarea fields in Settings?

Reply:
Yeah i'm not sure where that is coming from or why, but it sounds like we should update the higher level textarea styling

Reply:
Refactored on 3dc07011b1ad074f5f8ae0e0124bebd50a9895d8

Reply:
much better, thanks! 😄